vcpkg/commands/hash.md

```diff
@@ -0,0 +1,42 @@
+---
+title: vcpkg hash
+description: Reference for the vcpkg hash command. Computes the hash of a specified file.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 08/17/2023
+ms.prod: vcpkg
+---
+
+
+# vcpkg hash
+
+## Synopsis
+
+```console
+vcpkg hash <file-to-hash> [algorithm]
+```
+## Description
+
+Computes the hash of a specified file. By default, it computes the SHA-512 hash, but another algorithm can be optionally specified. The output will be the computed hash for the given file displayed directly on the console.
+
+>[!NOTE]
+>The command will exit with an error if:
+>- The specified file does not exist.
+>- The given algorithm is not supported or invalid.
+>- There are any other issues reading the file or computing the hash.
```

Comment:
We usually don't document failure cases. 
I'd suggest describing what each argument requires instead.

> The command accepts two arguments. 
> * The first argument, required, must be a valid path to a file. 
> * The second argument, optional, must be `SHA256` or `SHA512`. If no argument is provided, defaults to `SHA512`. The algorithm to use to calculate the hash of the file.